requirements.txt

```diff
@@ -12,3 +12,4 @@ pygments
 capstone==4.0.1
 enum34
 pytest
+testresources
```

Comment:
Why is this needed?

Reply:
Not sure if it's required, but it got rid of this message:
`ERROR: launchpadlib 1.10.13 requires testresources, which is not installed.  `

Reply:
We might want to move to install this only for ubuntu/debian installs via python3-testresources as in https://github.com/pypa/pip/issues/5372